tests/test_transforms.py

```diff
@@ -99,7 +99,6 @@ def test_elastic_transform_interpolation(monkeypatch, interpolation):
     [A.ShiftScaleRotate, {'rotate_limit': 45}],
     [A.RandomScale, {'scale_limit': 0.5}],
     [A.RandomSizedCrop, {'min_max_height': (80, 90), 'height': 100, 'width': 100}],
-    [A.RandomSizedBBoxSafeCrop, {'height': 100, 'width': 100}],
```

Comment:
Why did you remove this tests?

Reply:
After this commit, `bboxes` parameter is essential to run `RandomSizedBBoxSafeCrop`.  So, I tried to add any `bboxes` parameter to the test. But there are some augmentations not supporting `bboxes` in the test, so I just removed `RandomSizedBBoxSafeCrop`. If this is not a good way, please give me an advice.

Reply:
I see, this tests work only with images or images and masks. Add RandomSizedBBoxSafeCrop to this tests was a mistake.